terraform/testcases/jaeger_mock/parameters.tfvars

```diff
@@ -1,6 +1,6 @@
 validation_config = "jaeger-mocked-server-validation.yml"
 
-sample_app_image = "aottestbed/aws-otel-jz-sample-app:v0.1.4"
+sample_app_image = "public.ecr.aws/aws-otel-test/jaeger-zipkin-sample-app:v0.1.5"
 
 sample_app = "aws-otel-jz-sample-app"
```

Comment:
This `sample_app` name should probably be `jaeger-zipkin-sample-app` since that's the directory.

Reply:
It looks like it was changed in the `zipkin_mock`, but not in the `jaeger_mock`.